### ISSUE TITLE
Cannot install clippy

### ISSUE THREAD

**Issue by antoyo** [Manishearth/rust-clippy#945]
Hello.
I recently switched to using `rustup` so I decided I would try the `clippy` sub-command but I am unable to install `clippy`.

Here are the steps I used:

```bash
~ rustup default nightly
info: using existing install for 'nightly-x86_64-unknown-linux-gnu'
info: default toolchain set to 'nightly-x86_64-unknown-linux-gnu'

  nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.10.0-nightly (9c6904ca1 2016-05-18)

~ cargo -V
cargo 0.11.0-nightly (fb03dad 2016-05-18)

~ cargo install clippy > result
```
(see [attached result.txt file](https://github.com/Manishearth/rust-clippy/files/273619/result.txt))

I use ArchLinux and I installed `rustup` using this command:

```bash
yaourt -S rustup
```

Is this a clippy or a rustup issue?

How can I install `clippy` using rustup?

Thanks for your help.

**Comment by mcarton** [comment]
Clippy does not build on today’s nightly.

Ref https://github.com/Manishearth/rust-clippy/pull/944.

**Comment by mcarton** [comment]
Published new 0.0.69 version.
